ngraph/src/ngraph/op/non_max_suppression.hpp

```diff
@@ -182,6 +182,60 @@ namespace ngraph
                 int64_t max_boxes_output_from_input() const;
             };
         } // namespace v3
+
+        namespace v4
+        {
+            /// \brief NonMaxSuppression operation
+            ///
+            class NGRAPH_API NonMaxSuppression : public op::v3::NonMaxSuppression
```

Comment:
Should there be `evaluate` method?

Reply:
It will be there as soon as it will appear in v3::NMS (main part), shape value propagation should be added later.